Jenkinsfile

```diff
@@ -7,9 +7,9 @@ class Globals {
     // constants
     static final String PROJECT = 'pyflexplot'
     static final String IMAGE_REPO = 'docker-intern-nexus.meteoswiss.ch'
-    static final String IMAGE_NAME = 'docker-intern-nexus.meteoswiss.ch/flexpart-cosmo/pyflexplot'
+    static final String IMAGE_NAME = 'docker-intern-nexus.meteoswiss.ch/numericalweatherpredictions/dispersionmodelling/pyflexplot'
 
-    static final String AWS_IMAGE_NAME = 'mch-meteoswiss-dispersionmodelling-flexpart-cosmo-pyflexplot-repository'
+    static final String AWS_IMAGE_NAME = 'numericalweatherpredictions/dispersionmodelling/pyflexplot'
```

Comment:
sorry, we need to keep deploying to the aws ecr repo: `mch-meteoswiss-dispersionmodelling-flexpart-cosmo-pyflexplot-repository` for flexpart-cosmo. Can we set this based on the choice of `awsAccount`?